lib/instance-validator.js

```diff
@@ -112,7 +112,7 @@ class InstanceValidator {
           .then(newError => { throw newError || error; }))
       )
       .then(() => runHooks('afterValidate', this.modelInstance, this.options))
-      .return(this.modelInstance);
+      .return();
```

Comment:
is this method's return value returned elsewhere? This must introduce a breaking change at a different end